packages/mux-uploader/test/web-test-runner.config.mjs

```diff
@@ -2,6 +2,22 @@ import { esbuildPlugin } from '@web/dev-server-esbuild';
 import { importMapsPlugin } from '@web/dev-server-import-maps';
 
 export default {
+  testFramework: {
+    config: {
+      ui: 'bdd',
+      timeout: '5000',
+    },
+  },
+  middleware: [
+    function respondWithUploadSuccess(context, next) {
+      if (context.url === '/mock-upload-endpoint') {
```

Comment:
suggestion(non-blocking): Add a code comment here referencing/discussing where/how it's used in the tests.